src/generator/pages.clj

```diff
@@ -116,10 +116,11 @@
   "Returns map of filename (eg. index.html) -> html"
   [m]
   (let [people-data (contentful/get-contentful :person-collection-query {:list false})
-        people (have vector? (get-in people-data [:personCollection :items]))]
+        people (have vector? (get-in people-data [:personCollection :items]))
+        filtered-people (filter #(get-in % [:createPersonPage]) people)]
```

Comment:
This works. but why not modify the graphql query with a where clause, so you don't load the persons into clojure to begin with?